src/app/components/preview/preview.component.ts

```diff
@@ -102,6 +103,10 @@ export class PreviewComponent extends PageComponent
   ngOnInit() {
     super.ngOnInit();
 
+    from(this.infoDrawerOpened$).subscribe(val => {
```

Comment:
does it need to be unsubscribed? I have a feeling it may leak subscriptions here